packages/ensnode-subgraph-graphql/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "ensnode-subgraph-graphql",
```

Comment:
let's name this package `ponder-subgraph-api` or `ponder-subgraph-compat` since that's all it will do, and we can eventually publish it as such or `@namehash/ponder-subgraph-api` instead of contributing it back to ponder directly